frontend/stylesheet.css

```diff
@@ -0,0 +1,309 @@
+/*
+HEADER FROM THIS POINT
+The Display at the very top of the website
+ */
+
+.header {
+    display: flex;
+    padding: 20px;
+    background: aquamarine;
+}
+
+.ourLogo {
+    flex-grow: 1;
+    font-size: xx-large;
+}
+
+.filler {
+    flex-grow: 0.63;
+    margin-top: 45px;
+}
+
+.sduLogo {
+    margin-top: 20px;
+}
+
+
+/*
+QUEUE TABLE FROM THIS POINT
+The logic behind the queue.
+ */
+
+.inputContainer {
+    display: flex;
+    height: 24em;
+
+}
+
+.queueTable {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    padding: 16px;
+    margin-right: 20px;
+
+
+}
+
+.progressAlignment {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+
+.arrowUp {
+    margin: 3px;
+    font-size: 26px;
+    color: dodgerblue;
+}
+
+.arrowDown {
+    margin: 3px;
+    font-size: 26px;
+    color: dodgerblue;
+}
+
+.close {
+    margin: 3px;
+    font-size: 24px;
+    color: red;
+}
+
+.batchProgress {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+    padding: 10px;
+}
+
+.beerQueueTable td, .beerQueueTable th {
+    border: 1px solid black;
+    text-align: left;
+    padding: 8px;
+}
+
+
+.beerQueueTable tr:nth-child(even) {
+    background-color: #dddddd;
+}
+
+/*
+STORAGE FROM THIS POINT
+This part keeps track of the quantity of Barley, Hops, Wheat, etc.
+ */
+
+.storageQuantity {
+    display: flex;
+    align-content: flex-end;
+    align-items: flex-end;
+    justify-content: flex-start;
+    width: 440px;
+    height: 12em;
+    overflow: hidden;
+    position: relative;
+    margin: 22px auto;
+    flex-grow: 1;
+
+
+}
+
+.storageAlt {
+    display: block;
+    flex-direction: column;
+    margin-top: 2px;
+
```

Comment:
Make the body margin 0, so that the page elements reach the whole page, look at top bar for example